docs/spec/channel.md

```diff
@@ -182,6 +182,9 @@ Each channel CRD MUST contain an array of subscribers:
 Each channel CRD MUST contain an array of subscribers:
 [`spec.subscribers`](https://github.com/knative/eventing/blob/master/pkg/apis/duck/v1beta1/subscribable_types.go)
 
+Note: the array of subscribers MUST NOT be set directly on the channel, but
+rather appended by the subscription itself.
```

Comment:
This means, we need to disable direct setting on IMC and other channels in the code as well.

Reply:
@aliok -- ok I've updated, please let me know what you think!